vespajlib/src/main/java/com/yahoo/text/JSON.java

```diff
@@ -1,15 +1,23 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.text;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 import java.util.Map;
+import java.util.Objects;
+
+import static com.yahoo.yolean.Exceptions.uncheck;
 
 /**
- * Static methods for working with the map textual format which is parsed by {@link MapParser}
```

Comment:
"The text format which name should not be spoken" 🤔

Reply:
what do you mean? the name is "map textual format", but this class is now more than just utils for that.

Reply:
Would be simpler with "Static methods for working with JSON" ;)